packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -25,6 +25,27 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
     },
     legendContainer: {
       paddingTop: '4px'
+    },
+    hoverCardTextStyles: {
+      fontFamily: 'Segoe UI',
+      fontSize: '12px',
+      lineHeight: '14px'
+    },
+    hoverCardDataStyles: {
+      color: legendColor === '' ? '#000000' : legendColor,
+      fontSize: '28px',
+      fontFamily: 'Segoe UI',
+      fontWeight: 'bold',
+      lineHeight: '31px'
+    },
+    hoverCardRoot: {
+      paddingLeft: '16px',
+      paddingRight: '22px',
+      paddingTop: '15px',
+      paddingBottom: '8px'
+    },
+    opacityChangeOnHover: {
+      opacity: isChartSelected ? (isSelected ? '' : '0.6') : ''
```

Comment:
This can be simplified to `(isChartSelected && isSelected) ? '0.6' : ''`

Reply:
It can be, will change it

Reply:
missed the styles file in the PR, will change it